saleor/graphql/schema.graphql

```diff
@@ -143,8 +143,8 @@ type AssignNavigation {
 
 type Attribute implements Node {
   id: ID!
-  productTypes(before: String, after: String, first: Int, last: Int): ProductTypeCountableConnection
-  productVariantTypes(before: String, after: String, first: Int, last: Int): ProductTypeCountableConnection
+  productTypes(before: String, after: String, first: Int, last: Int): ProductTypeCountableConnection!
+  productVariantTypes(before: String, after: String, first: Int, last: Int): ProductTypeCountableConnection!
```

Comment:
Is `!` expected?

Reply:
Yes, this is coming from graphene-django https://github.com/graphql-python/graphene-django/pull/610.